packages/react/src/components/header/Header.stories.tsx

```diff
@@ -303,3 +466,37 @@ export const WithSkipLink = (args) => {
     </Header>
   );
 };
+
+export const WithSkipLinkInDarkMode = (args) => {
+  return (
+    <Header {...args} theme="dark" onDidChangeLanguage={languageChangedAction}>
+      <Header.SkipLink skipTo="#content" label="Skip To Content" />
```

Comment:
On chrome: I think the SkipLink looks a bit funny with extra black border inside the box? Or is it in the designs? The border breaks on the right hand side too.
<img width="414" alt="Screenshot 2023-08-11 at 12 28 23" src="https://github.com/City-of-Helsinki/helsinki-design-system/assets/108321134/9ce4fea5-19cc-4b22-a854-825524ecc1f3">

This doesn't happen on Firefox or Safari though.

Reply:
Seems like chrome adds extra outline for `:focus-visible`. Fixed by adding
` 
 &:focus-visible {
    outline-offset: unset;
 }
`

Reply:
Chrome has default `:focus-visible` style. 
`a:-webkit-any-link:focus-visible {
    outline-offset: 1px;
}`